docs/rtd_requirements.txt

```diff
@@ -1,41 +1,57 @@
 -i https://pypi.python.org/simple
-amqp==2.5.1
-billiard==3.6.1.0
-boto3==1.9.216
-botocore==1.12.216
+amqp==2.6.1
```

Comment:
Overall LGTM, just curious, I haven't updated this file before, do you know why this file is updated?

Reply:
`docs/rtd_requirements.txt` is the legacy requirements file that is not used in the build of RBAC anymore due to the use of `pipenv`. I have been keeping it updated so Synk can scan the dependencies correctly, as it cannot currently scan `Pipfile` or `Pipfile.lock`